common.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+	getKeyvaluestoreEndpoint: function(){
+		if (process.argv.length < 3 && !process.env.MICRO_KEYVALUESTORE_ENDOPOINT) {	    
```

Comment:
Could this be rewritten so that we have checks in the following order (I think the priority is clear):

```
if (process.argv.length == 3)
else if process.env.MICRO_KEYVALUESTORE_ENDPOINT
else
```